zlibWrapper/Makefile

```diff
@@ -7,8 +7,9 @@
 
 # Paths to static and dynamic zlib and zstd libraries
 ifneq (,$(filter Windows%,$(OS)))
-STATICLIB = ../../zlib/libz.a ../lib/libzstd.a
-IMPLIB    = ../../zlib/libz.dll.a ../lib/libzstd.a
+ZLIBDIR = ../../zlib1.2.8
```

Comment:
I checked and MinGW/MSYS2 supports -lz so there is no need for "ifneq (,$(filter Windows%,$(OS))".

BTW, do we support CygWin compilation of zstd? It's installed on Travis so we use it.